src/os/bluestore/bluestore_types.h

```diff
@@ -1032,7 +1032,7 @@ WRITE_CLASS_DENC(bluestore_deferred_transaction_t)
 struct bluestore_compression_header_t {
   uint8_t type = Compressor::COMP_ALG_NONE;
   uint32_t length = 0;
-  int32_t zlib_winsize = -15;
+  int32_t compressor_message = -15;
```

Comment:
-15 makes no sense here. Better set to 0 by default.

Reply:
> -15 makes no sense here. Better set to 0 by default.

OK